src/index.ts

```diff
@@ -487,7 +488,11 @@ class NotificationAPI {
 
       const date = document.createElement('p');
       date.classList.add('notificationapi-notification-date');
-      date.innerHTML = timeAgo.format(new Date(n.date), 'round-minute');
+      const customStyle: FormatStyle = {
+        units: 'minute',
+        future: false
```

Comment:
If you change future=true it will not pass tests. Probably, as the documents says future is false by default. @sahandseifi